### PR TITLE
try going through the SCM

### PR DESCRIPTION
**Pull request by despair86**
i actually have no idea how to use the QT process control classes

### CODE REVIEW

File: src/process/WindowsLokinetProcessManager.cpp [hunk 1]
```diff
@@ -18,7 +18,7 @@ WindowsLokinetProcessManager::WindowsLokinetProcessManager()
 bool WindowsLokinetProcessManager::doStartLokinetProcess()
 {
     // try searching one level up from CWD
-    bool success = success = QProcess::startDetached("net start lokinet");
+    bool success = QProcess::startDetached("net start lokinet");
     if (success)
```
Comment (majestrate) [inline src/process/WindowsLokinetProcessManager.cpp]: `if(not success)`

Comment (despair86) [reply src/process/WindowsLokinetProcessManager.cpp]: right i got that mixed up with the plain old `system(3C)`

File: CMakeLists.txt [hunk 2]
```diff
@@ -2,11 +2,20 @@ cmake_minimum_required(VERSION 3.10)
...
     HOMEPAGE_URL https://github.com/loki-project/loki-network-control-panel
     LANGUAGES CXX)
+else()
+project(lokinet-gui
+    VERSION 0.3.0
+    DESCRIPTION "Lokinet graphical control panel"
+    HOMEPAGE_URL https://github.com/loki-project/loki-network-control-panel
+    LANGUAGES CXX RC)
+endif()
+
```
Comment (jagerman) [inline CMakeLists.txt:10]: I think you can do:
```cmake
if (WIN32)
    enable_language(RC)
endif()
```
for this.

File: src/version.rc [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+#ifndef APSTUDIO_INVOKED
+/////////////////////////////////////////////////////////////////////////////
+//
+// Generated from the TEXTINCLUDE 3 resource.
+//
+
+
+/////////////////////////////////////////////////////////////////////////////
+#endif    // not APSTUDIO_INVOKED
```
Comment (jagerman) [inline src/version.rc:113]: JFC Windows.  We get both XML *AND* this archaic stuff?  Joy.

Comment (despair86) [reply src/version.rc:113]: ha
it's what tells the runtime link editor to pop up a UAC prompt so we can do exits

Comment (despair86) [reply src/version.rc:113]: if you look over......

File: src/app.xml [hunk 4]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0" xmlns:asmv3="urn:schemas-microsoft-com:asm.v3">
+  <trustInfo xmlns="urn:schemas-microsoft-com:asm.v3">
+    <security>
+      <requestedPrivileges>
+        <requestedExecutionLevel level="requireAdministrator"/>
```
Comment (despair86) [inline src/app.xml:6]: ......here

File: src/process/WindowsLokinetProcessManager.cpp [hunk 5]
```diff
@@ -23,35 +18,15 @@ WindowsLokinetProcessManager::WindowsLokinetProcessManager()
...
-    {
-        // when upgraded from old installations (before 0.5.x), lokinet.exe would be one
-        // directory higher
-        success = QProcess::startDetached("..\\lokinet.exe");
-
-        if (! success)
-            qDebug("QProcess::startDetached() failed");
-    }
-    
+    bool success = QProcess::startDetached("net start lokinet");
```
Comment (majestrate) [inline src/process/WindowsLokinetProcessManager.cpp]: can we use the full path for this?

**Comment by despair86** [comment]
@jagerman @majestrate this should be ready now

**Pull request merged**